src/views/Comment.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <!-- <Sidebar class="sidebar" /> -->
     <main class="comment-page">
-        <h1>Comment</h1>
+        <h1>Comment pergepr</h1>
```

Comment:
can you convert it to the original version,  I am concerned it conflict with another branch (thun)